src/components/ha-button-toggle-group.ts

```diff
@@ -28,6 +28,8 @@ export class HaButtonToggleGroup extends LitElement {
 
   @property({ reflect: true }) size: "small" | "medium" = "medium";
 
+  @property({ type: Boolean, reflect: true }) public wrap = true;
```

Comment:
it is easier to turn this around, and call it `no-wrap` as setting an attribute to false is harder than adding a attribute to set it to true.